File: App2/src/app/_core/_model/farms/culling-tank.ts
export interface CullingTank {
  id: number;
  farmGuid: string;
  areaGuid: string;
  type: string;
  cullingTankNo: string;
  cullingTankName: string;
  cullingTankPrincipal: string;
  cullingTankLength: number | null;
  cullingTankWidth: number | null;
  comment: string;
  createDate: string | null;
  createBy: number | null;
  updateDate: string | null;
  updateBy: number | null;
  deleteDate: string | null;
  deleteBy: number | null;
  status: number | null;
  guid: string;
  cancelFlag: string;
  startDate: any | null;
  endDate: any | null;
}
export interface CullingTankScreen {
  id: number;
  cullingTankName: string;
  guid: string;
}

File: PigFarm/Models/CullingTank.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PigFarm.Models
{
    public partial class CullingTank
    {
        public decimal Id { get; set; }
        public string FarmGuid { get; set; }
        public string AreaGuid { get; set; }
        public string Type { get; set; }
        public string CullingTankNo { get; set; }
        public string CullingTankName { get; set; }
        public string CullingTankPrincipal { get; set; }
        public decimal? CullingTankLength { get; set; }
        public decimal? CullingTankWidth { get; set; }
        public string Comment { get; set; }
        public DateTime? CreateDate { get; set; }
        public decimal? CreateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public decimal? UpdateBy { get; set; }
        public DateTime? DeleteDate { get; set; }
        public decimal? DeleteBy { get; set; }
        public decimal? Status { get; set; }
        public string Guid { get; set; }
        public string CancelFlag { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
    }
}
